src/tracks/playabletrack/wavetrack/ui/SpectrumCache.cpp

```diff
@@ -573,7 +573,7 @@ bool WaveClipSpectrumCache::GetSpectrogram(const WaveClip &clip,
    // purposely offset the display 1/2 sample to the left (as compared
    // to waveform display) to properly center response of the FFT
    fillWhere(mSpecCache->where, numPixels, 0.5, correction,
```

Comment:
I think I wrote this elsewhere, bug again:  `0.5` should change to `true` at the commit "Make waveform display work again" to match the change of argument type.